packages/massa-web3/src/experimental/basicElements/mas.ts

```diff
@@ -19,12 +19,8 @@ export type Mas = bigint
  *
  * @throws An error if the value is not a safe integer.
  */
-export function fromMas(value: number): Mas {
-  if (!Number.isSafeInteger(value)) {
-    throw new Error(ERROR_NOT_SAFE_INTEGER)
-  }
-
-  return BigInt(value * POWER_TEN ** NB_DECIMALS)
+export function fromMas(value: U64): Mas {
```

Comment:
Here we return Mas type. Maybe it would be more clear if we return NanoMas

Reply:
I don't understand. The main type is the Mas, and currencies, to avoid precision issues, are mostly represented in their minor units. 

That being said, do you want me to add a comment on the Mas type to explain that it is represented in its minor unit, the nano massa?

Reply:
It might be useful to avoid misunderstanding